app/views/case_contacts/_form.html.erb

```diff
@@ -35,15 +35,15 @@
   <div id="contact-type-form" class="field contact-type form-group">
     <h2 id="contact-type-label"><%= form.label :contact_types %></h2>
 
-    <% contact_types = CaseContact::CONTACT_TYPES.each_with_index do |contact_type, index| %>
+    <% @contact_types.each_with_index do |contact_type, index| %>
       <div class="form-check">
-        <%= check_box_tag "case_contact[contact_types][]",
-          contact_type,
-          case_contact.contact_types&.include?(contact_type),
+        <%= check_box_tag "case_contact[case_contact_contact_type_attributes][][contact_type_id]",
```

Comment:
Is there a form helper we could use to simplify this?

Reply:
I tried a few helpers but I couldn't make them work. :sad:

I'm working on another PR now that have a similar form input, I'll give it another chance and if it end up making it work I'll update here.